api/python/setup.py

```diff
@@ -123,4 +123,5 @@
     },
     download_url="https://pypi.org/project/epam.indigo",
     test_suite="tests",
+    extras_require={"ml": ["scikit-learn", "pytest"]},
```

Comment:
We don't need pytest here, since it's test dependency, not runtime.

Reply:
Pytest moved to requirements_dev.txt